Wrappers/Python/cil/processors/PaganinPhaseProcessor.py

```diff
@@ -0,0 +1,387 @@
+# -*- coding: utf-8 -*-
+#  Copyright 2020 United Kingdom Research and Innovation
+#  Copyright 2020 The University of Manchester
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+# Authors:
+# CIL Developers, listed at: https://github.com/TomographicImaging/CIL/blob/master/NOTICE.txt
+
+from cil.framework import Processor
+import numpy as np
+
+from scipy.fft import fft2
+from scipy.fft import ifft2
+from scipy.fft import fftshift
+from scipy import constants
+
+from tqdm import tqdm
+
+import logging
+import dask
+from dask import delayed
```

Comment:
if `dask` is a new dependency you'll need to add it to:

- `recipe/meta.yaml`
- `scripts/create_local_env_for_cil_development.sh`
- `scripts/requirements-test.yml`